src/shape.rs

```diff
@@ -84,6 +115,7 @@ pub trait Shape: Sized {
     /// magnitude values are also possible when the shape is more complex.
     ///
     /// [`area`]: #tymethod.area
+    /// [winding number]: https://mathworld.wolfram.com/ContourWindingNumber.html
```

Comment:
Any particular reason why the mathworld link here? Wikipedia also seems good. I'm ok with either, though.

Reply:
No particular reason, we've used mathworld in other places and the explanations are concise.